service/worker/service.go

```diff
@@ -296,6 +307,13 @@ func NewConfig(dc *dynamicconfig.Collection, params *resource.BootstrapParams) *
 			dynamicconfig.WorkerPersistenceGlobalMaxQPS,
 			0,
 		),
+
+		StandardVisibilityPersistenceMaxReadQPS:   dc.GetIntProperty(dynamicconfig.StandardVisibilityPersistenceMaxReadQPS, 9000),
+		StandardVisibilityPersistenceMaxWriteQPS:  dc.GetIntProperty(dynamicconfig.StandardVisibilityPersistenceMaxWriteQPS, 9000),
+		AdvancedVisibilityPersistenceMaxReadQPS:   dc.GetIntProperty(dynamicconfig.AdvancedVisibilityPersistenceMaxReadQPS, 9000),
+		AdvancedVisibilityPersistenceMaxWriteQPS:  dc.GetIntProperty(dynamicconfig.AdvancedVisibilityPersistenceMaxWriteQPS, 9000),
```

Comment:
those default values seems way too high, but I realize those are the current default values used. :(